src/widgets/buttons.ts

```diff
@@ -534,7 +566,7 @@ export function button (dom: HTMLDocument, iconURI: string | undefined, text: st
  * @returns <dDomElement> - the button
  */
 export function cancelButton (dom: HTMLDocument, handler: (event: any) => void) {
-  return button(dom, cancelIconURI, 'Cancel', handler)
+  return button(dom, cancelIconURI, 'Cancel', handler, {})
```

Comment:
if you make options optional, then you don't need to add `, {}` here.

Reply:
good catch.  I changed it when I had made it required and didn't change it back.